docs/source/platform_technical_documentation/ade_custom_extensions/jobsui.ipynb

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
<p>View and Submit jobs icon is not visible in my version of ADE. Is it still there on other's version or has it been removed. In that case, this page needs to be modified. However, since we are going to make this version of RTD v2, this issue can also be fixed in the V3 version of RTD.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/165/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Looks good to me. Ready to merge.

Reply:
I think you'd only see this in the DIT version of the ADE. One option is we could revert https://github.com/MAAP-Project/maap-documentation/pull/144 , and reapply it after V2 tagging?

Reply:
Yes, reverting and reapplying should resolve this.